src/server.c

```diff
@@ -5768,6 +5768,7 @@ sds genValkeyInfoString(dict *section_dict, int all_sections, int everything) {
                 "redis_version:%s\r\n", REDIS_VERSION,
                 "server_name:%s\r\n", SERVER_NAME,
                 "valkey_version:%s\r\n", VALKEY_VERSION,
+                "rdb_version:%i\r\n", RDB_VERSION,
```

Comment:
One should ideally be able to look up the code from the `valkey_version` and determine the RDB version.

Further, the operator looking at this field needs to be aware that the server can load RDB files  whose version is lesser than the current server supported version. Hope this doesn't cause any confusion.